Makefile

```diff
@@ -35,6 +35,10 @@ generate-functions-ci:
 	go run github.com/openshift-knative/hack/cmd/prowgen --config config/kn-plugin-func.yaml $(ARGS)
 .PHONY: generate-functions-ci
 
+generate-event-ci:
```

Comment:
Can you rename this to `generate-kn-plugin-event-ci` ? We already have `generate-eventing-ci`. It's confusing

Reply:
The func plugin got `generate-functions-ci`, so I just followed that.

Maybe, we can change the function's target as well to have some consistency: `make generate-plugin-func-ci`

The `make generate-kn-plugin-event-ci` is so mouthful!

BTW. this convention is stuttering - `make generate-`. Better would be `make event-plugin`, `make eventing` etc.

Reply:
At minimum let's just rename your new target to generate-plugin-event-ci